test/hello_world.test.js

```diff
@@ -1,120 +0,0 @@
-var test = require('tape');
```

Comment:
Yes, would be great to add a class example back. But, I would put adding an example of an `async standalone` function higher priority in your list than a class example (partly because the latter is more complex and less frequently a need).